src/Common/src/Internal/Cryptography/BCryptAlgorithmCache.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+
+using Microsoft.Win32.SafeHandles;
+using NTSTATUS = Interop.BCrypt.NTSTATUS;
+using BCryptOpenAlgorithmProviderFlags = Interop.BCrypt.BCryptOpenAlgorithmProviderFlags;
+
+namespace Internal.Cryptography
```

Comment:
That'd be my preference as well.  Eventually we want all of this interop stuff to end up in src\Common\src\Interop\Windows\libraryname\etc.